locale/en/messages.json

```diff
@@ -109,42 +109,6 @@
       ]
     ]
   },
-  "1 Backend": {
```

Comment:
That's a lot of deletion..

Reply:
We should find a better solution for storing these, it seems to add quite a lot of accidental complexity

Reply:
I run `lingui-extract-with-plugins` and this is what I get with no errors... @natmegs any idea? 🙏

Reply:
Oh, gotcha... some adjustments to the library should be made. Very helpful wiki page!